dokka-subprojects/plugin-base/src/test/kotlin/model/InheritorsTest.kt

```diff
@@ -512,4 +512,58 @@ class InheritorsTest : AbstractModelTest("/src/main/kotlin/inheritors/Test.kt",
         }
     }
 
+    @Test
+    fun `DRI of generic inherited members (fake override) should lead to super member`() {
+        inlineModelTest(
+            """
+                |interface A<T> { fun x(): T }
+                |interface B : A<Int> { }
+                """
+        ) {
+            with((this / "inheritors" / "A" / "x").cast<DFunction>()) {
+                dri.classNames equals "A"
+                dri.packageName equals "inheritors"
+            }
+            with((this / "inheritors" / "B" / "x").cast<DFunction>()) {
+                dri.classNames equals "A"
+                dri.packageName equals "inheritors"
+            }
+        }
+    }
+    @Test
+    fun `DRI of inherited members should lead to super member`() {
+        inlineModelTest(
+            """
+                |interface A { fun x() = 0 }
+                |interface B : A { }
```

Comment:
should we add additional test with `interface` which overrides `fun`?

AFAIU there is a matrix of variants here:
* interface vs class (vs open/abstract class, not sure it makes a difference)
* fake-override vs override vs no-override

May be it will be good to add all those cases?

Reply:
There is no difference between interface and class (in general - `classlikes`) for our logic here.
I would prefer to not add these tests  (interface vs class) since we have a lot of tests where interfaces and classes are interchangeable with the same result.